### PR TITLE
Addded fix for LCW closeChat methods throws exception

### PR DESCRIPTION
**Pull request by nikhilwaichal-96**
### Description
getVoiceVideoCalling() In omnichannel sdk file was throwing exception when user closes chat.

## Solution Proposed
Added fix to this issue by adding isVoiceVideoCallingEnabled() in omnichannel sdk and when it is enabled we will be calling  getVoiceVideoCalling() method in omnichannel chat widget.

### Acceptance criteria
When user close chat it shouldn't throw an exception about a "Feature disabled".

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

https://github.com/microsoft/omnichannel-chat-widget/assets/128818969/bcc196df-d036-4c16-8808-d9008e6084ac

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/common/endChat.ts [hunk 1]
```diff
@@ -210,7 +210,13 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
...
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk;
+        if(chatSDK.isVoiceVideoCallingEnabled()) {
+            voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        }
+        else{
+            return;
+        }
         if (voiceVideoCallingSdk && voiceVideoCallingSdk.isInACall()) {
```
Comment (xTEddie) [inline chat-widget/src/components/livechatwidget/common/endChat.ts:220]: Will the widget crash if  `voiceVideoCallingSdk` was `undefined`? Should `voiceVideoCallingSdk` be nullable?

Comment (nikhilwaichal-96) [reply chat-widget/src/components/livechatwidget/common/endChat.ts:220]: Fixed comment marked voiceVideoCallingSdk  as null

**Review by mithunjohny — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/endChat.ts [hunk 2]
```diff
@@ -210,7 +210,13 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
 export const endVoiceVideoCallIfOngoing = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>) => {
     let callId = "";
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk=null;
+        if(chatSDK.isVoiceVideoCallingEnabled()) {
+            voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        }
+        else{
```
Comment (xTEddie) [inline chat-widget/src/components/livechatwidget/common/endChat.ts:217]: `else` section looks redundant and may not be needed.

File: chat-widget/src/components/livechatwidget/common/endChat.ts [hunk 3]
```diff
@@ -210,7 +210,13 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
 export const endVoiceVideoCallIfOngoing = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>) => {
     let callId = "";
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk=null;
```
Comment (xTEddie) [inline chat-widget/src/components/livechatwidget/common/endChat.ts:213]: nit: spacing around `=`

**Review by mithunjohny — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/endChat.ts [hunk 4]
```diff
@@ -210,7 +210,10 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
 export const endVoiceVideoCallIfOngoing = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>) => {
     let callId = "";
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk = null;
+        if (chatSDK.isVoiceVideoCallingEnabled()) {
```
Comment (elopezanaya) [inline chat-widget/src/components/livechatwidget/common/endChat.ts:214]: I think all the logic for end chat should be inside this block, if voice is not enabled then next block is irrelevant as well

**Review by matan-hershberg-nuance — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]

**Pull request merged**